builddefs/generic_features.mk

```diff
@@ -15,6 +15,7 @@
 
 SPACE_CADET_ENABLE ?= yes
 GRAVE_ESC_ENABLE ?= yes
+TRI_LAYER_ENABLE ?= yes
```

Comment:
This feature is disabled by default on purpose, and was added to the featureset enabled by default for VIA <https://github.com/qmk/qmk_firmware/blob/develop/builddefs/common_features.mk#L623>
```suggestion
```